src/app/navigation/examples/application-launcher-example.component.html

```diff
@@ -0,0 +1,61 @@
+<ul class="nav navbar-nav">
```

Comment:
Please rename the 'examples' directory as 'example'.

Also, please move the component and example under a directory, named src/app/navigation/application-launcher